chaosazure/vmss/fetcher.py

```diff
@@ -6,18 +6,33 @@
 from logzero import logger
 
 from chaosazure import init_client
-from chaosazure.rgraph.resource_graph import fetch_resources
+from chaosazure.common.resources.graph import fetch_resources
 from chaosazure.vmss.constants import RES_TYPE_VMSS
 
 
+def fetch_instances(scale_set, instance_criteria,
+                    configuration, secrets) -> List[Dict[str, Any]]:
+    if not instance_criteria:
+        instance = choose_vmss_instance_at_random(
+            scale_set, configuration, secrets)
+        result = [instance]
+
+    else:
+        result = choose_vmss_instance(
+            scale_set, configuration, instance_criteria, secrets)
+
+    return result
+
+
 def choose_vmss_instance_at_random(vmss_choice, configuration, secrets):
```

Comment:
can we add return type like we have in `fetch_instances`

Reply:
Solved